menas/ui/css/style.css

```diff
@@ -26,6 +26,18 @@ html, body {
   min-width: 40rem !important;
 }
 
+.minW30rem {
+  min-width: 30rem !important;
+}
+
+.maxH1000 {
+  max-height: 90% !important;
```

Comment:
I don't know if this is because I don't know CSS, but this seems confusing to me. I would read it as `max height 1000` and would infer that it is in `px` but this is `90%`